### ISSUE TITLE
can't compile ios build in github actions 

### ISSUE THREAD

**Issue by jay-athelas** [superlistapp/super_native_extensions#388]
Building ios with flutter errors in github actions, works in local but errors in the CI server from github actions

I already tried setting rust versions but nothing works 

`The following build commands failed:
2024-06-15T22:15:10.2055610Z              	PhaseScriptExecution [CP-User]\ Build\ SuperNativeExtensions\ Rust\ library /Users/runner/Library/Developer/Xcode/DerivedData/Runner-aixrnwwuerndpkeiqaoqortzrvbz/Build/Intermediates.noindex/ArchiveIntermediates/Runner/IntermediateBuildFilesPath/Pods.build/Release-iphoneos/super_native_extensions.build/Script-B60078EBBFDC64C400A6EDB63F400D8C.sh (in target 'super_native_extensions' from project 'Pods')`

i tried with this 
`rustc 1.70.0 (90c541806 2023-05-31)
ruby 3.0.7p220 (2024-04-23 revision 724a071175) [x86_64-darwin22]
1.[15](https://github.com/getathelas/scribe-mobile/actions/runs/9532706651/job/26275112675#step:9:16).2
Xcode 15.0.1
Build version 15A507
Dart SDK version: 3.3.0 (stable) (Tue Feb 13 10:25:19 2024 +0000) on "macos_x64"`

and changed rust version multiple times up and down, what is going on here ? we are blocked this packaged is being added as a transitive dependency by flutter quill.

https://pub.dev/packages/flutter_quill/changelog

I appreciate any guidance here, because I am out of ideas

**Comment by knopp** [comment]
Can you post here verbose build logs? This does not contain the actual error.

**Comment by jay-athelas** [comment]
sure here is the complete log
https://jmp.sh/s/QDv09yzJbXh8gj9bB3AQ

**Comment by jay-athelas** [comment]
I think this is the error ?

```
SEVERE: error[E0282]: type annotations needed for `Box<_>`
2024-06-16T05:17:53.0338530Z              SEVERE:   --> /Users/runner/.cargo/registry/src/index.crates.io-6f17d22bba15001f/time-0.3.25/src/format_description/parse/mod.rs:83:9
2024-06-16T05:17:53.0338670Z              SEVERE:    |
2024-06-16T05:17:53.0338790Z              SEVERE: 83 |     let items = format_items
2024-06-16T05:17:53.0338880Z              SEVERE:    |         ^^^^^
2024-06-16T05:17:53.0338950Z              SEVERE: ...
2024-06-16T05:17:53.0339230Z              SEVERE: 86 |     Ok(items.into())
2024-06-16T05:17:53.0339740Z              SEVERE:    |              ---- type must be known at this point
2024-06-16T05:17:53.0339940Z              SEVERE:    |
2024-06-16T05:17:53.0340310Z              SEVERE: help: consider giving `items` an explicit type, where the placeholders `_` are specified
2024-06-16T05:17:53.0340380Z              SEVERE:    |
2024-06-16T05:17:53.0340560Z              SEVERE: 83 |     let items: Box<_> = format_items
2024-06-16T05:17:53.0340730Z              SEVERE:    |              ++++++++
2024-06-16T05:17:53.0340810Z              SEVERE: 
2024-06-16T05:17:53.0340950Z              SEVERE:    Compiling form_urlencoded v1.2.0
2024-06-16T05:17:53.0341060Z              SEVERE:    Compiling rand_chacha v0.3.1
2024-06-16T05:17:53.0341340Z              SEVERE: For more information about this error, try `rustc --explain E0282`.
2024-06-16T05:17:53.0349610Z              SEVERE: error: could not compile `time` (lib) due to 1 previous error
2024-06-16T05:17:53.0350090Z              SEVERE: warning: build failed, waiting for other jobs to finish...
2024-06-16T05:17:53.0350400Z              SEVERE: --------------------------------------------------------------------------------
2024-06-16T05:17:53.0350590Z              SEVERE: #0      runCommand (package:build_tool/src/util.dart:118:5)
2024-06-16T05:17:53.0350800Z              SEVERE: #1      RustBuilder.build (package:build_tool/src/builder.dart:139:5)
2024-06-16T05:17:53.0350910Z              SEVERE: <asynchronous suspension>
2024-06-16T05:17:53.0351210Z              SEVERE: #2      ArtifactProvider.getArtifacts (package:build_tool/src/artifacts_provider.dart:68:25)
2024-06-16T05:17:53.0351320Z              SEVERE: <asynchronous suspension>
2024-06-16T05:17:53.0351570Z              SEVERE: #3      BuildPod.build (package:build_tool/src/build_pod.dart:31:23)
2024-06-16T05:17:53.0351720Z              SEVERE: <asynchronous suspension>
2024-06-16T05:17:53.0352000Z              SEVERE: #4      BuildPodCommand.runBuildCommand (package:build_tool/src/build_tool.dart:48:5)
2024-06-16T05:17:53.0352110Z              SEVERE: <asynchronous suspension>
2024-06-16T05:17:53.0352370Z              SEVERE: #5      BuildCommand.run (package:build_tool/src/build_tool.dart:34:5)
2024-06-16T05:17:53.0352470Z              SEVERE: <asynchronous suspension>
2024-06-16T05:17:53.0352710Z              SEVERE: #6      CommandRunner.runCommand (package:args/command_runner.dart:212:13)
2024-06-16T05:17:53.0352810Z              SEVERE: <asynchronous suspension>
2024-06-16T05:17:53.0353000Z              SEVERE: #7      runMain (package:build_tool/src/build_tool.dart:248:5)
2024-06-16T05:17:53.0363560Z              SEVERE: <asynchronous suspension>
2024-06-16T05:17:53.0364440Z              SEVERE: --------------------------------------------------------------------------------
2024-06-16T05:17:53.0364840Z              SEVERE: BuildTool arguments: [build-pod, ../rust, super_native_extensions]
```

**Comment by jay-athelas** [comment]
also found this link 
https://users.rust-lang.org/t/time-crate-compilation-error/111789

**Comment by knopp** [comment]
This should be fixed in current super_clipboard version (0.8.17).

**Comment by jay-athelas** [comment]
Thanks @knopp overriding this dependencies seems to have worked, wondering why in my local didn't complain about this issue?
